types/compare.go

```diff
@@ -35,6 +37,50 @@ func CompareUint64(x, y uint64) int {
 	return 1
 }
 
+//VecComparUU returns []int64 comparing the []uint64 x to []uint64 y
+func VecComparUU(x, y []uint64, res []int64) {
```

Comment:
`VecComparUU(x, y []uint64, res []int64, n)` ?

Reply:
I had refined it. Thank you ~